exercises/practice/parallel-letter-frequency/.docs/instructions.append.md

```diff
@@ -0,0 +1,5 @@
+# Instructions append
+
+## Using Parallelism
+
+The goal of this exercise is to learn to implement parallelism. In emacs lisp this is achived by using [`asynchronous processes`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Asynchronous-Processes.html#:~:text=An%20asynchronous%20process%20is%20controlled,%2Dtype%20(see%20below)). You may also want to look at the documentation for [`batch mode`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Batch-Mode.html), [`sentinels`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Sentinels.html) and [`receiving output from processes`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Output-from-Processes.html).
```

Comment:
```suggestion
The goal of this exercise is to learn to implement parallelism.
In Emacs Lisp this is achieved by using [`asynchronous processes`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Asynchronous-Processes.html#:~:text=An%20asynchronous%20process%20is%20controlled,%2Dtype%20(see%20below)). 
You may also want to look at the documentation for [`batch mode`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Batch-Mode.html), [`sentinels`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Sentinels.html) and [`receiving output from processes`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Output-from-Processes.html).
```

For Exercism docs, we do one line per sentence. I'm not sure if this is documented on the site though.  We may also want to say the goal is to practice parallelism with Emacs Lisp rather than implementing that concept. The later is a much broader concept and outside the scope of a single exercise.

Reply:
done. I think I got it right.